homework/week1/RosmaryFC/HoroscopeHW20150508/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,152 @@
+<resources>
+    <string name="app_name">HoroscopeHW20150508</string>
+
+    <string name="title_section1">Home</string>
+    <string name="title_section2">Signs</string>
+    <string name="title_section3">Type your Birthday</string>
+    <string name="title_section4">Game</string>
+
+    <string-array name="signs_array">
+        <item>Capricorn</item>
+        <item>Aquarius</item>
+        <item>Pisces</item>
+        <item>Aries</item>
+        <item>Taurus</item>
+        <item>Gemini</item>
+        <item>Cancer</item>
+        <item>Leo</item>
+        <item>Virgo</item>
+        <item>Libra</item>
+        <item>Scorpio</item>
+        <item>Sagittarius</item>
+    </string-array>
+
+    <string name="capricorn">
```

Comment:
One thing you might want to utilize is \n to go to a new line in your string, and <b></b> for bold. It will make your text look much nicer when running the app